analyzere/resources.py

```diff
@@ -31,7 +31,15 @@ def profile(self):
 # Exchange rate tables
 
 class ExchangeRateTable(DataResource):
-    pass
+    def currencies(self):
+        path = '%s/currencies' % self._get_path(self.id)
```

Comment:
```suggestion
        path = '%s/currencies' % self._get_path(self.id)
        path = '{}/currencies'.format(self._get_path(self.id))
```
I think we generally use `.format` or you could use a new fancy `f`string.
would look something like path = f'{self._get_path(self.id)}/currencies'

Reply:
Updated as suggested.